main.py

```diff
@@ -1,9 +1,11 @@
 import typer
 
 from todo import Todo
+from store import PersistentDb
 
 app = typer.Typer()
-todo = Todo()
+db = PersistentDb("DATABASE_URL")
+todo = Todo(db)
```

Comment:
what I love that the Todo class below just worked "as is", only had to update the thing we pass in here, "abstractions" :)